### ISSUE TITLE
Bugs with NM using aoe Charm attack

### ISSUE THREAD

**Issue by w3genesis** [LegionDark/Issues#1542]
**Date & Time**: 2017-6-28

**FFXI Client Version (use `/ver`)**: 30170427_0

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: yes

**Character Name**: Gouken

**Nation**: Bastok

**Job(level)/Sub Job(level)**: 99 Sam/dnc

**NPC or Monster or item Name**: Giltine

**Zone name**: Eldieme Necropolis (S)

**Coordinates (use `$where`)**: X:-4 Y:1 Z:44 (36)

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: Yes

**Steps To Reproduce / any other info**: 
Giltine uses Gala Macabre (aoe charm) just as my white mage Miniakuma is rising up from reraise, Miniakuma is charmed and is alive with hp, but is in the face down position(looks dead), will move with Giltine, and cannot become uncharmed even after 5 minutes.
In other words, the charm effect lasts forever when hit by aoe charm during reraise animation.

Another issue with Giltine, when it is in low hp, it has high regain and can do the infinite loop of aoe charm -> walk around with charmed party -> instantly aoe charm again once party is uncharmed -> repeat, making the last 20% extremely hard to fight. Aoe charm and high regain does not mix.

**Comment by TeoTwawki** [comment]
Informed DarkstarProject of charm issues, disabling the move in question until this is fixed.

**Comment by w3genesis** [comment]
Thanks Teo. Had similar problems fighting Pil and Rani that did aoe charm as well, but Giltine was the worst since he spammed death spell between aoe charm so someone is always dead and reraising.

**Comment by w3genesis** [comment]
Went to fight Pil just now, it still uses Enthrall (aoe charm) multiple times during the fight. @TeoTwawki

**Comment by TeoTwawki** [comment]
Enthrall disabled.

**Comment by stevomatic** [comment]
I just started zilart path and this is happening to me with Holy Moly and his aoe charm, Soothing Aroma

**Comment by TeoTwawki** [comment]
not so Soothing Aroma also disabled.

**Comment by w3genesis** [comment]
Wait until you get to Morta :D that thing has 2 aoe charm moves!

**Comment by TeoTwawki** [comment]
> Wait until you get to Morta :D that thing has 2 aoe charm moves!

**You must _name names_!** 
So's I can temp kill all the AoE ones.


![izser 1](https://user-images.githubusercontent.com/6871475/28655497-58f2f5f8-726a-11e7-9f2b-c8d7b34ca48e.jpg)
![name-names21 1](https://user-images.githubusercontent.com/6871475/28655515-652cbdb8-726a-11e7-9e3a-47c0f0039862.jpg)
